src/feltorShw/feltor_hpc.cu

```diff
@@ -25,6 +26,27 @@
         density fields are the real densities in XSPACE ( not logarithmic values)
 */
 
+
+// Global variable to store the NC file id for the signal handler
+namespace ns_ncid
+{
+    int ncid;
+}
+
+void sigterm_handler(int signal)
+{
+    file :: NC_Error_Handle err;
+    std::cout << "sigterm_handler, got signal " << signal << std::endl;
+    std::cout << "ncid = " << ns_ncid :: ncid << std::endl;
+    if(ns_ncid :: ncid != -1)
```

Comment:
könnte man an der Stelle  noch abfragen ob das file überhaupt geöffnet ist, weil in der Hauptschleife in feltor_hpc wird das file immer wieder geöffnet und geschlossen. Ich habs mal ausprobiert und bekomm einen netcdf Fehler weil das file schon zu war glaub ich. 
Mpi Version funktioniert gut